Source/SocketExtensions.swift

```diff
@@ -31,7 +31,21 @@ enum JSONError : Error {
 
 extension Array {
     func toJSON() throws -> Data {
-        return try JSONSerialization.data(withJSONObject: self, options: JSONSerialization.WritingOptions(rawValue: 0))
+        #if os(iOS)
+        if #available(iOS 11.0, *) {
```

Comment:
You should be able to specify multiple things here. `#available(iOS 11.0, macOS 10.13, *)`